scripts/prettier.js

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
People say you can use thus plugin https://github.com/zertosh/eslint-plugin-prettify instead of this file now